src/app/thankYou/thankYou.tpl.html

```diff
@@ -12,7 +12,8 @@ <h2 class="panel-name" translate>
                   </h2>
 
                   <p translate>
-                    We are processing your gift now. We'll send an email confirmation to to <strong>{{$ctrl.email}}</strong> as well as a tax-deductible receipt by postal mail.
+                    We are processing your gift now. We'll send an email confirmation to to
```

Comment:
I noticed that there are 2 `to`s here. Could you fix that since you touched this line?